odoo/addons/base/wizard/base_language_install_views.xml

```diff
@@ -2,39 +2,53 @@
 <odoo>
     <data>
 
+        <record id="base.language_install_view_form_lang_switch" model="ir.ui.view">
+            <field name="name">Switch to language</field>
+            <field name="model">base.language.install</field>
+            <field name="priority">100</field>
+            <field name="arch" type="xml">
+                <form string="Switch to language">
+                    <group>
+                        <span class="o_form_label">
+                            <strong><field name="first_lang_id" readonly="True" options="{'no_open': True}"/></strong> has been successfully installed.
+                            Users can choose their favorite language in their preferences.
+                        </span>
+                    </group>
+                    <footer>
+                        <button name="reload" string="Close" type="object" class="btn-primary" data-hotkey="q"/>
+                        <button name="switch_lang" type="object" class="btn-primary ml-1" data-hotkey="w">
+                            Switch to <field name="first_lang_id" class="text-white" style="pointer-events: none;"/> &amp; Close
+                        </button>
+                    </footer>
+                </form>
+           </field>
+        </record>
+
         <record id="view_base_language_install" model="ir.ui.view">
             <field name="name">Load a Translation</field>
             <field name="model">base.language.install</field>
+            <field name="priority">50</field>
```

Comment:
Note that you don't necessarily have to change it as its default value is 16 AFAIR